workspace/spec/lita/handlers/teamcity_spec_anki.rb

```diff
@@ -195,6 +195,78 @@
       expect(output_string).to eq(expected_output)
     end
   end
+  
+  describe '#format_time' do
```

Comment:
This method does not mention in ticket. So could you help me show reason or remove it from PR

Reply:
I resloved conflict and edit this. this is not changed